source/NVDAObjects/window/scintilla.py

```diff
@@ -70,9 +70,10 @@ def _getOffsetFromPoint(self,x,y):
 		return watchdog.cancellableSendMessage(self.obj.windowHandle,SCI_POSITIONFROMPOINT,x,y)
 
 	def _getPointFromOffset(self,offset):
+		location = self.obj.location

```

Comment:
You're basically converting client to screen coordinates here, doing it manually. Is there a specific reason why you aren't using clientToScreen here? Does it fail?

Reply:
Hi Leonard,

You are right! I just forgot about it. This way makes it more explicit.